p2p/conn/connection.go

```diff
@@ -45,13 +45,19 @@ const (
 	defaultSendTimeout         = 10 * time.Second
 	defaultPingInterval        = 60 * time.Second
 	defaultPongTimeout         = 45 * time.Second
+	defaultMsgRecvQueueSize    = 1000
```

Comment:
Have we picked this size based on some testing/analysis?

Reply:
This was a guess. I have been running a mainnet follower since with a higher limit and will be looking at the high water mark to get an idea of a reasonable default value.

Reply:
@JeancarloBarrios please update this to 5000